src/react-currency-masked-input.jsx

```diff
@@ -79,3 +79,5 @@ CurrencyMaskedInput.defaultProps = {
   value : null
 }
 
+module.exports = CurrencyMaskedInput;
```

Comment:
got it. i remember seeing chatter about how this is *broken* in babel at the moment.